libs/common/src/errors/document-not-created.error.test.ts

```diff
@@ -0,0 +1,28 @@
+import { DocumentNotCreatedError } from './document-not-created.error';
+import { DatabaseError } from './database.error';
+
+describe('DocumentNotCreatedError', () => {
+  it('exposes the message the error was created with', () => {
```

Comment:
these its should start with "should" as in it('should....

Reply:
We have 4651 tests which do not begin with should and 4943 with do -- I think if we want to align these it's a different PR